src/dispatch/search/service.py

```diff
@@ -19,12 +21,13 @@ def get_by_name(*, db_session, name: str) -> Optional[SearchFilter]:
     return db_session.query(SearchFilter).filter(SearchFilter.name == name).first()
 
 
-def match(*, db_session, filter_spec, model, model_id: int):
-    """Matches an incident with a given search filter."""
-    model_cls = get_class_by_tablename(model)
+def match(*, db_session, filter_spec, class_instance):
+    """Matches an class instance with a given search filter."""
+    table_name = class_instance.sa_instance_state.mapper.mapped_table.name
```

Comment:
@kevgliss I have to test this. If you know a better way, let m know.

Reply:
I think we only have the reverse right now (strings to models). Here:

https://github.com/Netflix/dispatch/blob/master/src/dispatch/database.py#L65